src/groups/mqb/mqbs/mqbs_inmemorystorage.cpp

```diff
@@ -118,12 +118,9 @@ int InMemoryStorage::configure(BSLA_UNUSED bsl::ostream& errorDescription,
 
 void InMemoryStorage::setConsistency(const mqbconfm::Consistency& value)
 {
-    BALL_LOG_WARN_BLOCK
-    {
-        if (value.isStrongValue()) {
-            BALL_LOG_OUTPUT_STREAM << "Trying to configure strong consistency "
-                                   << "for in-memory storage";
-        }
+    if (value.isStrongValue()) {
```

Comment:
This is a rare code path and I don't think it's worth to add PREDICT_UNLIKELY hints